src/users/users.ts

```diff
@@ -102,7 +105,6 @@ export class Users {
       serializeAuthenticateUserWithTokenOptions({
         ...payload,
         clientSecret: this.workos.key,
-        grantType: 'authorization_code',
```

Comment:
I moved the `grantType` down into the serializer since it never varies, and also updated the types to be the associated string literal rather than `string`.